docs/user_guides/fs/vector_similarity_search.md

```diff
@@ -0,0 +1,96 @@
+---
```

Comment:
Can you add a link from this page https://docs.hopsworks.ai/latest/concepts/mlops/opensearch/ to this guide?